pom.xml

```diff
@@ -112,6 +112,12 @@
             <version>0.0.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
don't add this as a dependency. you will get this dependency via the nucleus. Update the nucleus dependency to 2.10.0-SNAPSHOT.